power/nut-client/nut-client.yaml

```diff
@@ -0,0 +1,47 @@
+name: nut-client
+depends:
+  - service: cri
+  - network:
+    - addresses
+    - connectivity
+    - etcfiles
+container:
+  entrypoint: ./upsmon
+  args:
+    - -F
+  mounts:
+    - source: /lib
+      destination: /lib
+      type: bind
+      options:
+        - bind
+        - ro
+    - source: /usr/lib
+      destination: /usr/lib
+      type: bind
+      options:
+        - bind
+        - ro
+    # config via .machine.files
+    - source: /var/etc/nut
+      destination: /usr/local/etc/nut
+      type: bind
+      options:
+        - bind
+        - ro
+    # /sbin/init talks to apid
+    - source: /system/run/apid/apid.sock
+      destination: /system/run/apid/apid.sock
+      type: bind
+      options:
+        - rshared
+        - rbind
+        - rw
+    # symlinked to /sbin/poweroff
+    - source: /sbin/init
+      destination: /sbin/init
+      type: bind
+      options:
+        - bind
+        - ro
+restart: always
```

Comment:
doesn't `upsmon` need a `rw` place to store state, like a socket or a pid file? or it doesn;t need one?

Reply:
it will continue without it

```
$ ./tc 1 logs ext-nut-client
192.168.40.1: Network UPS Tools upsmon 2.8.0
192.168.40.1: fopen /var/run/upsmon.pid: No such file or directory
192.168.40.1: Could not find PID file to see if previous upsmon instance is already running!
192.168.40.1:
192.168.40.1: UPS: ***@***.***:3493 (secondary) (power value 1)
192.168.40.1: writepid: fopen /var/run/upsmon.pid: No such file or directory
```

if you do make `/var/run` writeable, it complains about PID 1 being invalid and ignores it.